tests/cockpit-tests

```diff
@@ -16,21 +16,21 @@ if [ $loop = "yes" ]; then
 fi
 
 # Make sure directory is writable
-sudo mkdir -p /cache/github /cache/images
-sudo chown user:user /cache /cache/github /cache/images
+mkdir -p /cache/github /cache/images
 
 # HACK: /sys is read-only in containers. And libvirt tries
 # to write there, but only if a specific subtree exists here
-sudo -n mount -o bind /run /sys/devices/virtual/net/ || true
+if [ -d /sys/devices/virtual/net/lo ]; then
+    sudo -n mount -o bind /run /sys/devices/virtual/net/ || true
+fi
```

Comment:
To clarify, this will only actually run if you manually run a container on e. g. your own laptop. In OpenShift /net already gets bind-mounted to an empty directory, so this hack isn't necessary there, right?

Reply:
Right.